demisto_sdk/commands/common/tools.py

```diff
@@ -809,6 +809,7 @@ def get_file(
     type_of_file: Optional[str] = None,
     clear_cache: bool = False,
     return_content: bool = False,
+    keep_order: bool = True,
```

Comment:
why default=`True`? It's slower and doesn't matter in almost all cases

Reply:
because many UT fail of it

Reply:
Let's open a task for this, then a followup task to fix the UTs

Reply:
OK, I'll open now